tests/test_hidpp20.py

```diff
@@ -39,3 +39,43 @@ def device(self, hidpp, fd):
         yield dev
 
         hidpp.hidpp20_device_destroy(dev)
+
+    # IRoot
+    def test_root_get_feature(self, client, id, hidpp, device):
+        feature_index = hidpp.new_uint8_tp()
+        feature_type = hidpp.new_uint8_tp()
+        feature_version = hidpp.new_uint8_tp()
+
+        # IFeatureSet
+        assert not hidpp.hidpp_root_get_feature(device, 0x0001, feature_index, feature_type, feature_version)
```

Comment:
github isn't friendly but - aren't we using a `unittest.TestCase` as baseclass here? if so, we should use `self.assertNotEqual` etc. instead of plain asserts.

Reply:
We aren't, I tried to use it in the past but it would get messed up with pytest IIRC. I could try again though.

Reply:
please do, it's the reference unittesting class and pytest is explicitly a wrapper around unittest. so anything that doesn't work is more likely a bug in our code than an issue with pytest.

Reply:
We had a deeper look at pytest in an other project, and we now agree that relying on `unittest.TestCase` would mess things up with pytest.